doc/developer.texinfo

```diff
@@ -2649,20 +2777,23 @@ make distribution-check
 
 @need 1000
 @item
-Save the previous release in the history archives.
-
-@example
-make save-current-release
-@end example
+Once you have successfully reached this point, you have a valid release candidate
+and are ready to make the actual release.
+@strong{Caution:} If somewhere above you made a change to correct a problem you should have
+restarted the release process.
 
-[Time: moments]
+At this point we are done building and testing the release candidate and you
+should exit the Bash command shell created in step 12 and return to your
+original Bash shell.
 
 
 @need 1000
 @item
-Update the website.
+Update the website by moving the current release to the history
```

Comment:
Does this step still move the current release to the history archives?  I thought that "make staging" also copied the release there (at least in its own copy of the history).